### DISCUSSION TITLE
Issue in Flow past cylinder tutorial at High Re

### DISCUSSION THREAD

**Discussion by pratyush507** [fppimenta/rheoTool#98] [General]
When i am trying to run flow past cylinder at high Re by changing the density of the fluid i am getting same values as obtained in case of Re=0.01 (nearly 0)  as per the tutorial. At Re 100 i tried to plot the streamlines there was creeping flow and no recirculation regions are formed which is impossible. Can you help me figure out why i am getting wrong drag coefficient values at higher Re in rheoFoam solver??

**Comment by fppimenta** [comment]
Please read the user guide before opening issues. https://github.com/fppimenta/rheoTool/discussions/64

**Reply by pratyush507** [reply]
Even if i changed the scheme of div(phi,U) from none to cubista then also the values of drag are comming wrong is there any scaling we have to add in the drag coefficient code.
